support-frontend/assets/pages/digital-subscriber-thank-you/digitalSubscriptionThankYou.tsx

```diff
@@ -106,8 +80,6 @@ export function DigitalSubscriptionThankYou(): JSX.Element {
 			'signIn',
 		),
 		'appDownloadKindle',
```

Comment:
Could we relabel this "appDownloadEditionsHeader", currently the presence of "appDownloadKindle" here instructs `thankYouModuleData` to render the "Download the Guardian Editions app" container, the inclusion of "kindle" in the name is a hangover from when this checkout was served at the `/kindle` route, "appDownloadEditionsHeader" feels like a more relevant/accurate description of it's purpose now.

Reply:
Yes, makes sense, have renamed 'kindle' references to 'editions' throughout the thank-you components.